docs/guide/common_addons.md

```diff
@@ -71,4 +71,155 @@ dashboard accessible from the outside via `https` on port `443`.
 
 ![FIP](https://raw.githubusercontent.com/sapcc/kubernikus/master/assets/images/docs/containers/kubernetes/loadbalancer2.png)
 
-~> This setup exposes a unauthenticated Dashboard with full access to the cluster. This is a security risk. See the [Access Control](https://github.com/kubernetes/dashboard/wiki/Access-control) documentation for more info.
+~> This setup exposes an unauthenticated Dashboard with full access to the cluster. This is a security risk. See the [Access Control](https://github.com/kubernetes/dashboard/wiki/Access-control) documentation for more info.
+
+## Private Docker Registry in Kubernikus
+You can create a private docker registry in your Kubernikus cluster to store your Docker images. 
+
+### How it works
+The private registry runs as a Pod in your cluster. A proxy on each node is exposing a port onto the node (via a hostPort), which Docker accepts as "secure", since it is accessed by localhost.
+
+### Create a persisten volume claim
+There is already a default storageClass and your cluster knows that storage exists. You just have to create a persistent volume claim to claim the storage. 
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: registry-storage
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
1g seems too small.

Reply:
True. I'd change it to 5g + add to the description that the value should be adjusted regarding the desired size and available quota.